script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -267,7 +267,7 @@ So then why... Why are corpses with their blood sucked dry still appearing...?!
 -- Page 109, Offset 16841. Glued. Mods: @x.
 -- 相手が美人のお姉さんなら吸われちゃってもいいんだがねぇ」
 // Leading space for glue
- Although I guess I wouldn't mind having my blood sucked if the vampire was a beautiful older woman."
+ Although I guess I wouldn't mind having my blood sucked if the vampire was a beautiful onee-san."
```

Comment:
I think revert this change and the next
I'm for honorifics being maintained because they encode extra information which is missing in English. I don't think there's anything gained in this case.

Reply:
I really don't like the translation of 'onee-san' as 'older-woman' since that has a much different (to my knowlage) connotation in english. Given our audience is people who read visual novels I think that 'onee-san' is the better choice.

Reply:
I might side with Ben on this one, I think there are a couple times Arihiko's preference for a more mature type comes up - 
```
Day 3/03_00_ARC03_3.txt-[sha:d6281cfd00da2f05ac19f0869912a3074900faf2]{
Day 3/03_00_ARC03_3.txt--- Page 162, Offset 4362.
Day 3/03_00_ARC03_3.txt--- 「眼鏡の似合う上級生だって言うんだろ？」
Day 3/03_00_ARC03_3.txt:"You're saying you like older women who look good in glasses, right?"
Day 3/03_00_ARC03_3.txt-}
```
I think it's referenced again re: Noel when taking to Arihiko, but I can't find the line

Reply:
Ah, here it is
```
_ 635 -- おまえ、包容力のある年上が好きなんだろ？　良かったな、好み通りの25歳だ」
  636  You like older women who are patient, don't you? Good for you. She's 25, exactly how you like it."
```